pkg/addon/policyframework/manifests/managedclusterchart/templates/role_binding.yaml

```diff
@@ -1,18 +1,24 @@
 # Copyright Contributors to the Open Cluster Management project
 
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
+kind: RoleBinding
```

Comment:
This was changed to a role binding so that the permissions are more tightly scoped, especially in hosted mode where there could be multiple policy framework deployments on a single hosting cluster.